types/react-native-web.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference path="./react-native.d.ts" />
+
+declare module "react-native-web" {
+  export = React
+}
```

Comment:
hah, nice one

for those not following along, this basically redirects react-native-web to re-use the imports from react-native